src/core/auth/qgsauthmanager.cpp

```diff
@@ -111,15 +112,28 @@ QSqlDatabase QgsAuthManager::authDatabaseConnection() const
   if ( isDisabled() )
     return authdb;
 
-  QString connectionname = QStringLiteral( "authentication.configs" );
-  if ( !QSqlDatabase::contains( connectionname ) )
+  QString connectionName = QStringLiteral( "authentication.configs" );
+  // Starting with Qt 5.11, sharing the same connection between threads is not allowed.
```

Comment:
Hate to be pendantic -- can you change this comment to 

  // sharing the same connection between threads is not allowed

Because it's not a qt 5.11 thing